sql/pg_duckdb--0.3.0--1.0.0.sql

```diff
@@ -64,7 +64,9 @@ LANGUAGE C;
 
 DROP FUNCTION duckdb.install_extension(TEXT);
 CREATE FUNCTION duckdb.install_extension(extension_name TEXT, source TEXT DEFAULT 'core') RETURNS void
+    SECURITY DEFINER
     SET search_path = pg_catalog, pg_temp
+    SET duckdb.force_execution = false
```

Comment:
I assume it is the case - but will that be automatically reset after execution of the function?